### DISCUSSION TITLE
What does TrigPitch do in DualTM?

### DISCUSSION THREAD

**Discussion by amelishko** [djphazer/O_C-Phazerville#101] [Q&A] [answered]
Hello,

Can you please explain what does TrigPitch do in DualTM, 1.8 firmware? It looks like it combines short trigget + pitch information in one signal, but I can't come up with ideas on where do use it. 

Do I understand it correctly? What's the use case for it?

Thank you for the amazing firmware, really appreciate your work.
Mick

**Comment by djphazer** [comment, marked as answer]
Hey there! Looks like I need to update the [DualTM man page](https://firmware.phazerville.com/DualTM) in the docs... Sounds like you understand it correctly: it's a pitch value, but with a pulse at the beginning for a spiky transient. With a bit of Slew, it's like a fast decay envelope with variable sustain.

Some of my suggested uses:
* Pitched percussion and other Pew-Pew type sounds when patched to a V/Oct or Frequency input of an oscillator or filter.
  - my 2hp MMF can be pinged on the FM input, which was my primary use case.
* Patch to the Level input of Plaits for varied velocity sustain with a sharp attack.
* Patch to any VCA for a similar effect.
* Get weird by transposing the quantizer up or down by a few octaves to offset the whole thing.

Let me know if you think of any other interesting patch ideas!
